substrate/frame/revive/src/storage.rs

```diff
@@ -356,6 +361,28 @@ impl<T: Config> ContractInfo<T> {
 	pub fn load_code_hash(account: &AccountIdOf<T>) -> Option<sp_core::H256> {
 		<ContractInfoOf<T>>::get(&T::AddressMapper::to_address(account)).map(|i| i.code_hash)
 	}
+
+	/// Returns the amount of immutable bytes of this contract.
+	pub fn immutable_bytes(&self) -> u32 {
```

Comment:
nit maybe we should name this field and fn `immutable_bytes_len`, it feels that you are asking for the actual bytes otherwise

Reply:
Well, the field name `bytes` with the type `u32` kind of implies it. But for consistency, we can rename it to `immutable_data_length`, since we talk about `immutable_data` when we mean the actual data. WDYT?

Reply:
I vote for `immutable_data_len`.

Reply:
Yeah renamed it to `immutable_data_len` :)